docs/build/arm-exception-handling.md

```diff
@@ -139,17 +139,17 @@ The epilogues for canonical functions follow a similar form, but in reverse and
 |-----------------|-----------------------------------|----------|------------|
 |6|*`Stack Adjust`*!=0 and *`EF`*==0|16/32|`add   sp,sp,#xx`|
 |7|*`R`*==1 and *`Reg`*!=7|32|`vpop  {d8-dE}`|
-|8|*`C`*==1 or (*`L`*==1 and *`H`*==0) or *`R`*==0 or *`EF`*==1|16/32|`pop   {registers}`|
-|9a|*`H`*==1 and *`L`*==0|16|`add   sp,sp,#0x10`|
-|9b|*`H`*==1 and *`L`*==1|32|`ldr   pc,[sp],#0x14`|
+|8|*`C`*==1 or (*`L`*==1 and (*`H`*==0 or *`Ret`* !=0)) or *`R`*==0 or *`EF`*==1|16/32|`pop   {registers}`|
+|9a|*`H`*==1 and (*`L`*==0 or *`Ret`*!=0)|16|`add   sp,sp,#0x10`|
+|9b|*`H`*==1 and *`L`*==1 and *`Ret`*==0|32|`ldr   pc,[sp],#0x14`|
```

Comment:
In the Ret==0 case, return is made by pop {pc}, so it can't be the case here where return is being achieved by ldr pc, ...